controllers/notification_controller.py

```diff
@@ -20,9 +19,11 @@ def post(self):
         target_account_id = self.request.get('account_id')
         if target_account_id == current_user_account_id:
             client_id = self.request.get('client_id')
+
             client = MobileClient.get_by_id(int(client_id), parent=ndb.Key(Account, current_user_account_id))
             if client is not None:
-                # This makes sure that the client actually exists and that this user owns it
-                NotificationHelper.send_ping(client)
+                from helpers.tbans_helper import TBANSHelper
+                TBANSHelper.ping(client)
```

Comment:
- [ ] We should be using the response of these calls to send back a success or error to clients